.importlinter

```diff
@@ -19,21 +19,19 @@ forbidden_modules =
     openpilot.tinygrad
 ignore_imports =
     # remove these
+    openpilot.selfdrive.car.interfaces -> openpilot.selfdrive.controls.lib.events
+
+    # these are okay
```

Comment:
tests using the car interfaces + real route data will live in openpilot for the time being, due to the use of LogReader, though the plan is to generate the CAN data fully in the future